sdk/python/tests/test_historical_retrieval.py

```diff
@@ -441,6 +441,14 @@ def test_historical_features_from_bigquery_sources(
             ],
         )
 
+        bq_temp_table_path = job_from_sql.to_bigquery()
```

Comment:
Can you as a test with `to_bigquery(dry_run=True)` please ?

Reply:
done